react/Menu.tsx

```diff
@@ -42,7 +44,7 @@ const Menu: StorefrontFunctionComponent<MenuSchema> = ({
       <MenuContext.Provider value={menuContext}>
         <nav>
           <ul
-            className={classNames('list flex pl0 mv0', {
+            className={classNames(`${styles.menuContainer} list flex pl0 mv0`, {
```

Comment:
You don't have to use string interpolation here, you could just do

```suggestion
            className={classNames(styles.menuContainer, 'list flex pl0 mv0', {
```